### DISCUSSION TITLE
Cannot archive individual card

### DISCUSSION THREAD

**Discussion by estellecomment** [community/community#14539] [Projects and Issues]
I can archive a whole column's cards, but not individual cards.

From what I see in previous discussions there was a menu (with "..." icon) on individual cards, that I don't see on my board. Has it disappeared ? Am I missing something ? 

I see : No menu on card : 
![image](https://user-images.githubusercontent.com/911434/162402253-296b7315-5465-4fd6-ba8d-b8f34ce0bfff.png)

I expect : a way to archive the card from the board page, without going to the issue's page.

Thank you !

**Comment by zelalSensgreen** [comment]
Same issue as well. But when you try to click the place where "..." icon should be. It is still there but it is hidden from view.
![image](https://user-images.githubusercontent.com/96177670/162428131-90c41d05-8b33-4c8c-9996-433d453f41dc.png)

I think it is a small bug because the button is disappeared but it is still active.

**Comment by estellecomment** [comment]
Indeed ! Invisible menu 😄 

![image](https://user-images.githubusercontent.com/911434/162431977-e5ab80e5-56f4-4037-8211-12d69f1025c5.png)
